content/blog/2010/2010-06-29-11th-international-free-software-forum-in-brazil.adoc

```diff
@@ -9,7 +9,7 @@
   - news
 :author: rtyler
 ---
-Last week, friend-of-Hudson https://leandron.wordpress.com/[Leandro Nunes] sent the following message to the `users` https://jenkins.io/content/mailing-lists[mailing list] regarding his upcoming talk on continuous integration and Hudson:
+Last week, friend-of-Hudson https://leandron.wordpress.com/[Leandro Nunes] sent the following message to the `users` link:/mailing-lists[mailing list] regarding his upcoming talk on continuous integration and Hudson:
```

Comment:
```suggestion
Last week, friend-of-Hudson https://leandron.wordpress.com/[Leandro Nunes] sent the following message to the `users` link:/content/mailing-lists[mailing list] regarding his upcoming talk on continuous integration and Hudson:
```

Reply:
This is an intended change, `/content/` is redirected.